packages/app/src/flame/examples/empty.ts

```diff
@@ -1,7 +1,7 @@
-import { tid, vid } from './util'
+import { tid, validateExample, vid } from './util'
 import type { FlameDescriptor } from '../transformFunction'
 
-export const empty: FlameDescriptor = {
+export const empty: FlameDescriptor = validateExample({
```

Comment:
it should not be necessary to validate static code. Only when loading from unknown sources.

Reply:
True, however, we need to see what is and what is not optional, validate in this sense not only check the structures which should obviously be correct, but also provide defaults and fallbacks through valibot schemas, so from code standpoint optional parameters are always set, but example can then be simpler - no need to write camera/render settings etc, so they default with validation.

Reply:
yeah makes sense